### PR TITLE
BZ#2065695: PAO - cnf-tests Documentation of PAO should not cover all cnf-tests as only the latency launchers are supported

### PR DESCRIPTION
**Pull request by aireilly**
For the Performance Addon Operator, cnf-tests docs should not cover all cnf-tests as only the latency launchers are supported. References to other tests have been removed.

Version(s):
main, 4.11, 4.12

Manual CP and adjustment will be required for 4.6-4.10.

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2065695

Preview:

http://file.emea.redhat.com/aireilly/BZ2065695/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html

### CODE REVIEW

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
 The tests run by the image are split into three different phases:
 
 * Simple cluster validation
 * Setup
 * End to end tests
 
-The validation phase checks that all the features required to be tested are deployed correctly on the cluster.
-
-Validations include:
+The latency tests require validations. The validation phase checks that all the features required to be tested are deployed correctly on the cluster. Validations include:
```
Comment (fedepaol) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:16]: Why do we even want to list the tests below? I'd just remove everything

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 2]
```diff
@@ -343,181 +317,46 @@ For example, to change the `CNF_TESTS_IMAGE` with a custom registry run the foll
...
-
-When deleting the performance profile, the machine config pool is modified and nodes are rebooted. After a new iteration, a new profile is created. This causes long test cycles between runs.
-
-To speed up this process, set `CLEAN_PERFORMANCE_PROFILE="false"` to instruct the tests not to clean the performance profile. In this way, the next iteration will not need to create it and wait for it to be applied.
-
-[source,terminal]
-----
-$ docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
-CLEAN_PERFORMANCE_PROFILE="false" registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e DISCOVERY_MODE=true NODES_SELECTOR=node-role.kubernetes.io/worker-cnf registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
```
Comment (fedepaol) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:359]: Is this section relevant to the latency tests?

**Comment by fedepaol** [comment]
Is the disconnected mode relevant to latency tests? If so, we should limit the namespaces we mirror images to. If not, we should remove the whole section.
Same for the SNO considerations, limiting the nodes, discovery mode, ginkgo parameters and all the rest. I don't know if they are relevant to latency tests, but if they are not they should be removed.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 3]
```diff
@@ -27,18 +24,37 @@ Validations include:
...
 
-The tests need to perform an environment configuration every time they are executed. This involves items such as creating SR-IOV node policies, performance profiles, or PTP profiles. Allowing the tests to configure an already configured cluster might affect the functionality of the cluster. Also, changes to configuration items such as SR-IOV node policy might result in the environment being temporarily unavailable until the configuration change is processed.
+[IMPORTANT]
+====
+When running the latency tests, **always** run the tests with `DISCOVERY_MODE=true`. For example:
+
+[source,terminal]
+----
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e
+DISCOVERY_MODE=true registry.redhat.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
```
Comment (fedepaol) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:39]: This will run the full set of tests. We should be specific about latency tests

Comment (aireilly) [reply modules/cnf-performing-platform-verification-latency-tests.adoc]: I've updated this and the rest. We'll need to walk through this on a call I think.

**Comment by aireilly** [comment]
> Is the disconnected mode relevant to latency tests? If so, we should limit the namespaces we mirror images to. If not, we should remove the whole section.
> 
> Same for the SNO considerations, limiting the nodes, discovery mode, ginkgo parameters and all the rest. I don't know if they are relevant to latency tests, but if they are not they should be removed.

Martin wanted to keep disconnected info.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 4]
```diff
@@ -0,0 +1,777 @@
...
+-e KUBECONFIG=/kubeconfig/kubeconfig -e IMAGE_REGISTRY=registry.redhat.io/openshift4/ \
+-e CNF_TESTS_IMAGE=cnf-tests-rhel8:v{product-version} /usr/bin/mirror \
+--registry "my.local.registry:5000/" --images "/kubeconfig/images.json" \
+|  oc image mirror -f -
+----
+
+[id="cnf-performing-end-to-end-tests-running-in-single-node-cluster_{context}"]
+== Running latency tests on a {sno} cluster
+
+You can runn latency tests on {sno} clusters.
```
Comment (kquinn1204) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:223]: Looks like typo here runn

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 5]
```diff
@@ -0,0 +1,756 @@
+// Module included in the following assemblies:
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-performing-platform-verification-latency-tests_{context}"]
+= Performing latency tests for platform verification
+
+You can use the link:https://quay.io/repository/openshift-kni/cnf-tests?tag=latest&tab=tags[Cloud-native Network Functions (CNF) tests image] to run latency tests on a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed. The latency tests allows you to validate node tuning for your workload.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:8]: quay should not be used in the documentation for images of red-hat products. please use the official one instead.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 6]
```diff
@@ -0,0 +1,756 @@
...
+[id="discovery-mode_{context}"]
+== About discovery mode
+
+Discovery mode allows you to validate the functionality of a cluster without altering its configuration. Existing environment configurations are used for the tests. The tests attempt to find the configuration items needed and use those items to execute the tests. If resources needed to run a specific test are not found, the test is skipped, providing an appropriate message to the user. After the tests are finished, no cleanup of the pre-configured configuration items is done, and the test environment can be immediately used for another test run.
+
+Some configuration items are still created by the tests. These are specific items needed for a test to run; for example, an SR-IOV Network. These configuration items are created in custom namespaces and are cleaned up after the tests are executed.
+
+[IMPORTANT]
+====
+When running the latency tests, **always** run the tests with `-e DISCOVERY_MODE=true`. If you do not run the latency tests in discovery mode, your existing live cluster performance profile configuration will be modified.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:35]: it is recommended to run with discovery mode, but running the tests without it will:
- fail the tests if there is no relevant PP for the worker-cnf nodes
- use the existing PP 
code reference: https://github.com/openshift-kni/performance-addon-operators/blob/master/functests/4_latency/latency.go#L88
I believe this note was relevant for the other functional tests and not the latency ones.
@Tal-or @MarSik  please confirm/ correct me if I am wrong

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 7]
```diff
@@ -0,0 +1,756 @@
...
+If you want to override the default worker pool, pass the `-e ROLE_WORKER_CNF=<custom_worker_pool>` variable to the command specifying an appropriate label.
+====
+
+[id="cnf-measuring-latency_{context}"]
+== Measuring latency
+
+The `cnf-tests` image uses three tools to measure the latency of the system:
+
+* `hwlatdetect`
+* `cyclictest`
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:54]: supported only for > 4.8

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 8]
```diff
@@ -0,0 +1,756 @@
...
+|The variable specifies the maximum acceptable hardware latency in microseconds for the workload and operating system. If you do not set the value of `HWLATDETECT_MAXIMUM_LATENCY` or `MAXIMUM_LATENCY`, the tool compares the default expected threshold (20μs) and the actual maximum latency in the tool itself. Then, the test fails or succeeds accordingly.
+
+|`CYCLICTEST_MAXIMUM_LATENCY`
+|The variable specifies the maximum latency in microseconds that all threads expect before waking up during the `cyclictest` run. If you do not set the value of `CYCLICTEST_MAXIMUM_LATENCY` or `MAXIMUM_LATENCY`, the tool skips the comparison of the expected and the actual maximum latency.
+
+|`OSLAT_MAXIMUM_LATENCY`
+|The variable specifies the maximum acceptable latency in microseconds for the `oslat` test results. If you do not set the value of `OSLAT_MAXIMUM_LATENCY` or `MAXIMUM_LATENCY`, the tool skips the comparison of the expected and the actual maximum latency.
+
+|`MAXIMUM_LATENCY`
+|This is a unified variable you can apply for all the available latency tools.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:92]: better: "This is a unified variable to specify the maximum acceptable latency in microseconds applicable for all available latency tools."

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 9]
```diff
@@ -0,0 +1,756 @@
...
+
+|====
+
+[NOTE]
+====
+A variable that is specific to certain tests has precedence over the unified variable.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:98]: better: 
"A variable that is specific to a certain latency tool has precedence over the unified variable. So if OSLAT_MAXIMUM_LATENCY is set to 30 microseconds and MAXIMUM_LATENCY is set to 10 microseconds, the comparison of the actual maximum latency is done vs OSLAT_MAXIMUM_LATENCY."

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 10]
```diff
@@ -0,0 +1,756 @@
...
+[source,terminal,subs="attributes+"]
+----
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
+-e DISCOVERY_MODE=true registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
+/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+----
++
+[IMPORTANT]
+====
+You must run the latency tests with `-e DISCOVERY_MODE=true`. The latency tests can change the configuration of your cluster if you do not run in Discovery mode.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:121]: repeated note, see above comments.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 11]
```diff
@@ -0,0 +1,756 @@
...
+. Optional: Append `-ginkgo.v` to run the tests with increased verbosity.
+
+. Optional: To run the latency tests against a specific performance profile, run the following command, substituting appropriate values:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
+-e LATENCY_TEST_RUN=true -e LATENCY_TEST_RUNTIME=600 -e OSLAT_MAXIMUM_LATENCY=20 \
+-e PERF_TEST_PROFILE=<performance_profile> registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
+/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\[config\]|\[performance\]\ Latency\ Test"
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:135]: please change ginkgo focus value to :
"\[performance\]\ Latency\ Test"

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 12]
```diff
@@ -0,0 +1,756 @@
...
+/usr/bin/test-run.sh -ginkgo.v -ginkgo.focus="hwlatdetect"
+----
++
+`hwlatdetect` runs for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs).
++
+[NOTE]
+====
+For valid results, the test should run for at least 12 hours.
+====
++
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:174]: Let's move this note to the beginning of the latency section, this is an important one and valid for all tools.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 13]
```diff
@@ -0,0 +1,756 @@
...
+
+[...]
+
+• Failure [283.574 seconds]
+[performance] Latency Test
+/remote-source/app/vendor/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/4_latency/latency.go:62
+  with the hwlatdetect image
+  /remote-source/app/vendor/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/4_latency/latency.go:228
+    should succeed [It]
+    /remote-source/app/vendor/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/4_latency/latency.go:236
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:195]: Note that this is relevant for 4.11/4.12 releases, where the tests exist in NTO. for other versions please consider using the old command outputs as per each version documentation considering the availability of the tools in each version (hwlat and cyclictest are available since 4.9)

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 14]
```diff
@@ -0,0 +1,756 @@
...
+[id="cnf-performing-end-to-end-tests-running-cyclictest"]
+=== Running cyclictest
+
+The `cyclictest` tool measures the real-time kernel scheduler latency on the specified CPUs.
+
+.Prerequisites
+
+* You have logged into `registry.redhat.io` with your Customer Portal credentials.
+* You have installed the real-time kernel in the cluster.
+* You have applied a cluster performance profile by using Node Tuning Operator.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:314]: same note regaing versions, this is relevant only for 4.11 and above. for <4.11 we use Performance addon operator

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 15]
```diff
@@ -0,0 +1,756 @@
...
+# Thread 15: 110059 155917
+----
+
+[id="cnf-performing-end-to-end-tests-running-oslat"]
+=== Running oslat
+
+.Prerequisites
+
+* You have logged into `registry.redhat.io` with your Customer Portal credentials.
+* You have applied a cluster performance profile by using the Node Tuning Operator.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:438]: same here

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 16]
```diff
@@ -0,0 +1,756 @@
...
+
+.Procedure
+
+* To run the `hwlatdetect` tests, run the following command, substituting variable values as appropriate:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
+-e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf \
+-e LATENCY_TEST_CPUS=600 -e LATENCY_TEST_RUNTIME=600 \
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:163]: 600 is not a valid value for LATENCY_TEST_CPUS. this is an optional env var, you can remove it from the command.

Comment (shajmakh) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:163]: also, it doesn't match the output, the output matches a command that LATENCY_TEST_CPUS was not specified in it.

**Review by shajmakh — COMMENTED** [review]
@aireilly Thanks for the PR!
wrote a few comments in there. 
Please note that different versions need different command outputs and support different tools. see my comments inside for more details, and don't hesitate to reach out to me if something not clear.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 17]
```diff
@@ -0,0 +1,756 @@
...
+. You have applied all the required CNF configurations in the cluster.
+
+. You have a pre-existing `MachineConfigPool` CR applied in the cluster. The default worker pool is `worker-cnf`. For more information about creating the cluster performance profile, see _Provisioning a worker with real-time capabilities_.
+
+[id="discovery-mode_{context}"]
+== About discovery mode
+
+Discovery mode allows you to validate the functionality of a cluster without altering its configuration. Existing environment configurations are used for the tests. The tests attempt to find the configuration items needed and use those items to execute the tests. If resources needed to run a specific test are not found, the test is skipped, providing an appropriate message to the user. After the tests are finished, no cleanup of the pre-configured configuration items is done, and the test environment can be immediately used for another test run.
+
+Some configuration items are still created by the tests. These are specific items needed for a test to run; for example, an SR-IOV Network. These configuration items are created in custom namespaces and are cleaned up after the tests are executed.
```
Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:31]: not relevant sentence any more.

Comment (shajmakh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:31]: not relevant sentence any more.

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 5 above)
Comment (aireilly) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:8]: Updated.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 18]
```diff
@@ -0,0 +1,756 @@
...
+
+:_content-type: PROCEDURE
+[id="cnf-performing-platform-verification-latency-tests_{context}"]
+= Performing latency tests for platform verification
+
+You can use the link:https://quay.io/repository/openshift-kni/cnf-tests?tag=latest&tab=tags[Cloud-native Network Functions (CNF) tests image] to run latency tests on a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed. The latency tests allows you to validate node tuning for your workload.
+
+[IMPORTANT]
+====
+The `cnf-tests` image also includes several tests that are not supported by Red Hat at this time. Only the latency tests are supported.
```
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:12]: Are you sure? check first comment from Franck here: https://bugzilla.redhat.com/show_bug.cgi?id=2065695

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 6 above)
Comment (aireilly) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:35]: @MarSik Can you PTAL at this one please.

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 13 above)
Comment (aireilly) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:195]: Cool. Let's get this merged to main/enterprise-4.11+, and then I will manually CP and adjust for previous versions.

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 18 above)
[the remark this answers is not in the document]
Comment (shajmakh) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:14]: ok I see, then we can move on with this.

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 7 above)
Comment (shajmakh) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:56]: the current PR is applicable only to 4.11 and 4.12 (master - not yet released).

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 6 above)
Comment (MarSik) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:37]: This is going to customers. We want to be 100% sure they will not trash their cluster accidentally. Running without discovery mode and accidentally without focus will do that to them.

**Comment by shajmakh** [comment]
@aireilly thanks for the PR!
Looks good to me, except for one issue:
https://github.com/openshift/openshift-docs/pull/44823#discussion_r991356490

**Comment by aireilly** [comment]
> @aireilly thanks for the PR! Looks good to me, except for one issue: [#44823 (comment)](https://github.com/openshift/openshift-docs/pull/44823#discussion_r991356490)

Will update, thanks.

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 6 above)
Comment (aireilly) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:37]: "[IMPORTANT]
====
When running the latency tests, **always** run the tests with `-e DISCOVERY_MODE=true` and `-ginkgo.focus` set to the appropriate latency test. If you do not run the latency tests in discovery mode, your existing live cluster performance profile configuration will be modified by the test run.
===="

**Comment by aireilly** [comment]
@shajmakh Also added a note alongside each test: 

``` 
[IMPORTANT]
====
For valid results, the test should run for at least 12 hours.
====
```

This is true for all tests right?

**Comment by aireilly** [comment]
@MarSik Think we are nearly ready to go on this one finally, can you please take a final look/LGTM.

**Comment by aireilly** [comment]
@shajmakh should we update the latency test commands with realistic run times, ie 43200 seconds, not 600? Maybe doesn't matter ¯\(ツ)/¯

**Comment by shajmakh** [comment]
> @shajmakh should we update the latency test commands with realistic run times, ie 43200 seconds, not 600? Maybe doesn't matter ¯(ツ)/¯

we provide the commands as an example, it is too expensive to run the tests for that long time in addition that we don't have tuned clusters like customers and thus we can't anticipate good examples outputs. I believe the note you added regarding running the tests is worth mentioning.

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 19]
```diff
@@ -0,0 +1,763 @@
...
+====
+
+[id="cnf-performing-end-to-end-tests-running-cyclictest"]
+=== Running cyclictest
+
+The `cyclictest` tool measures the real-time kernel scheduler latency on the specified CPUs.
+
+.Prerequisites
+
+* You have logged into `registry.redhat.io` with your Customer Portal credentials.
```
Comment (abhatt-rh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:315]:
```suggestion
* You have logged in to `registry.redhat.io` with your Customer Portal credentials.
```

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 20]
```diff
@@ -0,0 +1,763 @@
...
+# Thread 14: 110060
+# Thread 15: 110059 155917
+----
+
+[id="cnf-performing-end-to-end-tests-running-oslat"]
+=== Running oslat
+
+.Prerequisites
+
+* You have logged into `registry.redhat.io` with your Customer Portal credentials.
```
Comment (abhatt-rh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:440]:
```suggestion
* You have logged in to `registry.redhat.io` with your Customer Portal credentials.
```

File: modules/cnf-performing-platform-verification-latency-tests.adoc [hunk 21]
```diff
@@ -0,0 +1,763 @@
...
+//
+// * scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-performing-platform-verification-latency-tests_{context}"]
+= Performing latency tests for platform verification
+
+You can use the Cloud-native Network Functions (CNF) tests image to run latency tests on a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed. The latency tests allows you to validate node tuning for your workload.
+
+The `cnf-tests` container image is available at `registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version}`.
```
Comment (abhatt-rh) [inline modules/cnf-performing-platform-verification-latency-tests.adoc:11]: You might want to check if the inline {product-version} attribute works without the `subs="attributes+"`

**Review by abhatt-rh — COMMENTED** [review]
Hi @aireilly,
The content mostly looks good. However, it could use significant modularization and restructuring, which can be attained by modifying the  `_topic_maps`. So, I added a few comments related to it. I understand that it might be rather huge endeavor, so feel free to track it as a separate issue. 
Let me know if you have any questions. Thanks! 

This PR could use a second pair of eyes, so feel free to request another peer review, if required.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/cnf-performing-platform-verification-latency-tests.adoc (hunk 21 above)
Comment (aireilly) [reply modules/cnf-performing-platform-verification-latency-tests.adoc:11]: It does :)

**Comment by mburke5678** [comment]
@aireilly I merged to main, 4.11, and 4.12. Your note above suggests we need _adjustments_ for earlier versions. How do we go about handling these changes?

**Comment by aireilly** [comment]
> @aireilly I merged to main, 4.11, and 4.12. Your note above suggests we need _adjustments_ for earlier versions. How do we go about handling these changes?

Basically I need to do a manual CP + adjust with QE cc @shajmakh :) Standby, will create those PRs presently.

**Comment by bergerhoffer** [comment]
@aireilly Just checking on these CPs. I see you have the one for 4.10 in progress - are you planning to wrap that one up first after getting QE, and then take that one to 4.9 and 4.8 after QE verifies?

**Comment by aireilly** [comment]
Yea that's the plan :)

**Comment by bergerhoffer** [comment]
Excellent, thanks!
